man/metab.Rd

```diff
@@ -52,7 +52,7 @@ Returns daily time series of gross primary production (GPP), respiration (R), an
 # fake data
 	datetime <- seq(as.POSIXct("2014-06-16 00:00:00", tz="GMT"),
      as.POSIXct("2014-06-17 23:55:00", tz="GMT"), length.out=288*2)
-	do.obs <- 2*sin(2*pi*(1/288)*(1:(288*2))+0.75*pi) + 8 + rnorm(288*2, 0, 0.5)
```

Comment:
yeah, pretty random. i figured the change would have to happen sometime, so just committed it. i guess it was @rBatt who didn't like the phase of the fake data? =)